03_simon_says/simon_says.rb

```diff
@@ -0,0 +1,3 @@
+def   repeat(var)
+  if
+end
```

Comment:
I think this needs to be on a separate PR. please remove the file from this branch :)